package.json

```diff
@@ -55,6 +55,7 @@
     "babel-jest": "24.9.0",
     "jest": "24.9.0",
     "prettier": "2.2.1",
+    "reflect-metadata": "0.1.13",
```

Comment:
What's the purpose of this dependency?

Reply:
I thought this was needed by jest for debugging in vs-code, but I removed it and debugging tests still works. It looks like this library isn't needed. Thanks!